ldk/go/readme.md

```diff
@@ -79,13 +75,13 @@ It can be useful to tail the log as you develop to see any errors, warnings, or
 ##### Linux
 
 ```sh
-tail -f /var/log/Sidekick/Sidekick-2020.08.12-1dcc37a.log
+tail -F /var/log/Sidekick/*.log
 ```
 
 ##### Mac
 
 ```sh
-tail -f ~/Library/Logs/Sidekick/Sidekick-2020.08.12-1dcc37a.log
+tail -F ~/Library/Logs/Sidekick/*.log
```

Comment:
I think this has changed to `~/Library/Logs/Olive\ Helps/`

Reply:
Gotcha - thanks for the heads up! I just changed it.